src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -634,53 +694,53 @@ private void executeCreateReferenceDatabases(long personID)
 	{
 		ReferenceDatabaseCreator creator = new ReferenceDatabaseCreator(this.dbAdapter, personID);
 
-		for (String key : this.referenceDatabasesToCreate.keySet())
+		for (Entry<String, Map<String, ?>> entry : this.referenceDatabasesToCreate.entrySet())
```

Comment:
Similar to above, it looks like the keys of the entry set are never used?  Could this be:
`for (Map<String, ?> refDB : this.referenceDatabasesToCreate.values())`